File: XJTUTOOLBOX_CMP/iosApp/iosApp/Adapters/KRExceptionAdapter.swift
import Foundation
import OpenKuiklyIOSRender

class KRExceptionAdapter: NSObject, IKRUncaughtExceptionHandlerAdapter {
    func uncaughtException(_ error: Error) {
        print("[KuiklyError] Uncaught exception: \(error.localizedDescription)")
    }
}

File: XJTUTOOLBOX_CMP/androidApp/src/main/kotlin/com/xjtu/toolbox/adapter/KRExceptionAdapter.kt
package com.xjtu.toolbox.adapter

import android.util.Log
import com.tencent.kuikly.core.render.android.adapter.IKRUncaughtExceptionHandlerAdapter

object KRExceptionAdapter : IKRUncaughtExceptionHandlerAdapter {
    override fun uncaughtException(throwable: Throwable) {
        Log.e("KuiklyError", "Uncaught exception", throwable)
    }
}
